LibTest/js_interop/JSExport/jsExport_A02_t04.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion You can either annotate specific instance members to only wrap
+/// those members or you can annotate the entire class, which will include all
+/// of its instance members.
+///
+/// @description Checks that it is a compile-time error to annotate with
+/// `JSExport()` an extension type instance members.
```

Comment:
Probably more readable:
```suggestion
/// @description Checks that it is a compile-time error to annotate extension
/// type instance members with `JSExport()`.
```